main/EventSystem.h

```diff
@@ -201,8 +201,9 @@ class CEventSystem : public CLuaCommon, StoppableTask, CNotificationObserver
 	std::string m_lua_Dir;
 	std::string m_szStartTime;
 
-	static const std::string m_szReason[], m_szSecStatus[];
-	static const _tJsonMap JsonMap[];
+	static const std::array<const char *, 7> m_szReason;
```

Comment:
If we rename m_szSecStatus to m_szEventsystemSecStatus , we could remove them from the header file.
(Same for the other array's)
The problem now/currently is that you need to specify the size on two locations (header and source file)
Declaring them only in the source file has also the benefit that no header needs to be changed and less source files need to be recompiled after a change
Is it possible to do this in the PR as well ?

Reply:
Doesn't look like it. It's being used in main/dzVents.cpp